runtime/lua/vim/lsp/buf.lua

```diff
@@ -345,7 +345,8 @@ local sig_help_ns = api.nvim_create_namespace('nvim.lsp.signature_help')
 --- @field silent? boolean
 
 --- Displays signature information about the symbol under the cursor in a
---- floating window.
+--- floating window. Allows cycling through signature overloads by mapping
+--- `<Plug>(CycleSignature)`.
```

Comment:
We should better scope the name here:
```suggestion
--- `<Plug>(LspCycleSignature)`.
```